frontend/packages/policy-editor/src/components/ExpandablePolicyCard/ExpandablePolicyElement/ExpandablePolicyElement.tsx

```diff
@@ -111,9 +114,9 @@ export const ExpandablePolicyElement = ({
         >
           <Label size='small'>{cardTitle}</Label>
           {isOpen ? (
-            <ChevronUpIcon title='Close the card' fontSize='1.8rem' />
+            <ChevronUpIcon title={t('"policy_editor.expandable_card_close_icon"')} fontSize='1.8rem' />
```

Comment:
Some double quotes should be removed. :)
```suggestion
            <ChevronUpIcon title={t('policy_editor.expandable_card_close_icon')} fontSize='1.8rem' />
```